src/hooks/useRichTextEditor.ts

```diff
@@ -0,0 +1,41 @@
+import { Extension, useEditor, Node } from '@tiptap/react';
+// @TODO: Investigate fix...
```

Comment:
Currently unable to import `Placeholder` component from tiptap due to an internal key naming conflict with tiptap core. #185